gprofiler/profilers/python_ebpf.py

```diff
@@ -201,7 +216,7 @@ def start(self) -> None:
         for f in glob.glob(f"{str(self.output_path)}.*"):
             os.unlink(f)
 
-        process = start_process(cmd)
+        process = start_process(cmd, tmpdir=self._pyperf_staticx_tmpdir)
```

Comment:
Does the directory have to exist prior to starting the process? If the directory does not already exist, does staticx create it?